keyboards/pimoroni/keybow2040/info.json

```diff
@@ -0,0 +1,116 @@
+{
+    "manufacturer": "Pimoroni",
+    "keyboard_name": "Keybow 2040",
+    "url": "https://shop.pimoroni.com/products/keybow-2040",
+    "maintainer": "fanf2",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "board": "GENERIC_RP_RP2040",
+    "features": {
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0x5069",
+        "pid": "0x4784"
+    },
+    "rgb_matrix": {
+        "driver": "IS31FL3731",
```

Comment:
change from IS31FL3731 to is31fl3731

Reply:
```suggestion
        "driver": "is31fl3731",
```

Reply:
This works after I rebase on top of current master